tests/unit/cli/cli-test.js

```diff
@@ -554,10 +554,10 @@ describe('Unit: CLI', function() {
     var help = stubValidateAndRun('help');
 
     return ember(['unknownCommand']).then(function() {
-      var errors = ui.errors.trim().split(EOL);
-      var helpfulMessage = /The specified command .*unknownCommand.* is invalid\. For available options/;
-      expect(errors[0]).to.match(helpfulMessage, 'expected an invalid command message');
-      expect(help.called, 'help command was executed').to.not.be.ok;
```

Comment:
the (old) lines 557-559 are equivalent to the (new) line 560
the (old) line 560 got lost -.- will add it to the `catch` block